### ISSUE TITLE
Custom property animation prevents glTF animations to be exporter

### ISSUE THREAD

**Issue by jjcasmar** [KhronosGroup/glTF-Blender-IO#1007]
In the default scene animate the default cube with a transform animation. This works perfectly. Now add a custom property and animate it. The exporter says
```
13:53:51 | WARNING: Animation 'CubeAction' could not be exported. Cause: Animations with target type '["prop"]' are not supported.
```
and the glTF file doesn't contain any animation, not even the transform one.

**Comment by julienduroure** [comment]
This is currently the intentional behaviour.
When some animation property can't be exported, we decided to not export animation at all. Choice was done to not export, instead of risking exporting an animation wrongly.

**Comment by jjcasmar** [comment]
The problem is that it also complicates to export EXT_property_animation. Right now a EXT_property_animation implementation would need to reimplement transform animations to be able to support a cube with a transform animation and a custom property animation
Is there any chance to recondir this?

**Comment by julienduroure** [comment]
Sure, let's keeping this ticket open to find a good solution

**Comment by jjcasmar** [comment]
#1022

**Comment by julienduroure** [comment]
Fixed  by 04122fe
